### PR TITLE
adds azure config e2e magic

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

**Review by tuommaki — APPROVED** [review]
LGTMIIWFY

**Review by kopiczko — APPROVED** [review]

File: pkg/chartvalues/apiextensions_azure_config_e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+			}),
+			errorMatcher: IsInvalidConfig,
+		},
+		{
+			name: "case 9: invalid .VersionBundleVersion",
+			config: newAPIExtensionsAzureConfigE2EConfigFromFilled(func(v *APIExtensionsAzureConfigE2EConfig) {
+				v.VersionBundleVersion = ""
+			}),
+			errorMatcher: IsInvalidConfig,
+		},
```
Comment (TheoBrigitte) [inline pkg/chartvalues/apiextensions_azure_config_e2e_test.go:192]: Would be good to also have a valid test case, for the sake of completeness.

**Review by TheoBrigitte — APPROVED** [review]
LGTM

File: pkg/chartvalues/apiextensions_azure_config_e2e_test.go (hunk 1 above)
Comment (xh3b4sd) [reply pkg/chartvalues/apiextensions_azure_config_e2e_test.go:192]: The valid test is above.

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**